### PR TITLE
Add Figure <number> to text and caption open-a-pr.md

### PR DESCRIPTION
**Pull request by chrismetz09**
Method described in [How to use captions](https://kubernetes.io/docs/contribute/style/diagram-guide/#how-to-use-captions).

Contributes to common method for referencing and labeling figures in docs.

### CODE REVIEW

File: content/en/docs/contribute/new-content/open-a-pr.md [hunk 1]
```diff
@@ -61,7 +61,7 @@ class tasks,tasks2 white
 class id1 k8s
 {{</ mermaid >}}
 
-***Figure - Steps for opening a PR using GitHub***
+Figure 1. Steps for opening a PR using GitHub***
```
Comment (chrismetz09) [inline content/en/docs/contribute/new-content/open-a-pr.md:64]: should be:
```
Figure 1. Steps for opening a PR using GitHub
```

File: content/en/docs/contribute/new-content/open-a-pr.md [hunk 2]
```diff
@@ -151,7 +151,8 @@ class 1,2,3,3a,4,5,6 grey
 class S,T spacewhite
 class changes,changes2 white
 {{</ mermaid >}}
-***Figure - Working from a local fork to make your changes***
+
+Figure 2. Working from a local fork to make your changes***
```
Comment (chrismetz09) [inline content/en/docs/contribute/new-content/open-a-pr.md:155]: should be:
```
Figure 2. Working from a local fork to make your changes
```

File: content/en/docs/contribute/new-content/open-a-pr.md [hunk 3]
```diff
@@ -379,7 +380,8 @@ classDef white fill:#ffffff,stroke:#000,stroke-width:px,color:#000,font-weight:b
 class 1,2,3,4,5,6,7,8 grey
 class first,second white
 {{</ mermaid >}}
-***Figure - Steps to open a PR from your fork to the K8s/website***
+
+Figure 3. Steps to open a PR from your fork to the K8s/website***
```
Comment (chrismetz09) [inline content/en/docs/contribute/new-content/open-a-pr.md:384]: should be:
```
Figure 3. Steps to open a PR from your fork to the K8s/website
```

**Comment by sftim** [comment]
Manual numbering is OK, but I would like to see a new web-development issue for using Hugo shortcodes to implement automatic numbering and linking. My worry is that otherwise we are increasing our overall maintenance burden for the site; the extra automation is aimed at making things scale better.

**Comment by kbhawkey** [comment]
Page preview:
https://deploy-preview-31652--kubernetes-io-main-staging.netlify.app/docs/contribute/new-content/open-a-pr/

**Comment by tengqm** [comment]
Agreed with Tim on this. Manual numbering will be difficult to maintain.

**Comment by reylejano** [comment]
Hi @chrismetz09 ,
The consensus is manual numbering will be difficult to maintain (I agree also).
I'm closing this PR but feel free to reopen this PR with `/reopen` if you feel otherwise or when there's a Hugo shortcode for automatic numbering and this PR can be revised to use automatic numbering

/close

**Comment by chrismetz09** [comment]
Full agreement that automation would be ideal. However I believe this PR should be merged. Reasons:
1. Already merged https://github.com/kubernetes/website/pull/31654 and https://github.com/kubernetes/website/pull/31653
2. Documented caption methods in [How to use caption](https://kubernetes.io/docs/contribute/style/diagram-guide/#how-to-use-captions). 
3. Diagram referral is embedded in content text. Absent an enhanced figure shortcode, authors conforming with the diagram guide will need to deal with manual numbers. Don't think this should be a scaling problem with just 2 or 3 diagrams per page likely.

IMHO, we should continue to encourage authors to use diagrams and employ the caption methodology described in the diagram guide. Then come up with new shortcode that handles caption, figure and referral.

**Comment by zacharysarah** [comment]
This seems useful enough that I think it's worth merging. I'd love to see correct use of figures become sufficiently pressing to require handling by shortcode.

/lgtm
/approve

**Pull request merged**